### ISSUE TITLE
RHEL8 DHCP Log streaming blank

### ISSUE THREAD

**Issue by scott80211** [Akkadius/glass-isc-dhcp#90]
I was able to get the install done with the help of "npm audit" during some failures.  I can log into the IP:3000 and for the most part seems to be working.  However when I click on Logs, and the DHCP Log shows up, regardless of what I type in the search, or leave it blank, and then click "start" it never shows anything in this window.

I've tried using "npm install console" and "npm install websocket", which downloaded and installed those, but it's still not showing anything.

Selinux has been disabled, even before the install.  I checked permissions, and changed/edited the glass_config.json file to point to the proper log and leases files.

All other screens seem to be showing fairly accurate information.  Leases can be searched on the active leases section.

I've even tried creating a sym link for /var/log/dhcp.log, in case the log file was being called some other way.

**Comment by madtempest** [comment]
I've never had logging working correctly, but are you actually seeing logs into /var/log/dhcp.log?

**Comment by scott80211** [comment]
Yes, and I have verified the settings for the Admin point to the
correct file.

**Comment by scott80211** [comment]
Via CLI/SSH or console, I can "tail -f /var/log/dhcp.log" and watch the log
fly by.

**Comment by wcarman1** [comment]
I know this is pretty old but I had to open port 8080 on the interface that this service is running on and restart the service. Then logging in the web ui started working.

**Comment by llima27** [comment]
@wcarman1 I'm trying to do the same. I'm a novice user running a small network at home. I've opened port 3000 on the firewall as per instructions here: https://github.com/Akkadius/glass-isc-dhcp/issues/62. Did you open port 8080 on the same firewall zone? Or was it something else? Thanks.

**Comment by wcarman1** [comment]
@llima27 sorry for the delay, I believe it was on the zone that you use to access the webserver. I believe for me it was public but it may be different for you though, it depends on your setup. I also moved to KEA/ Stork and away from ISC overall so I don't have my setup to look at anymore.

**Comment by llima27** [comment]
@wcarman1 No worries! I'll give it a try. I'll add both 8080 and 3000 to public to see if I can access the logs from the web.
